serde_valid/tests/complex_test.rs

```diff
@@ -135,6 +135,7 @@ struct TestInnerStruct<'a> {
     inner_vec_value: Vec<i32>,
 
     #[validate(minimum_duration = "15ms")]
+    #[validate(maximum_duration = "60ms")]
```

Comment:
Is it possible to have this notation ?

`#[validate(minimum_duration="15ms", maximum_duration="60ms"]`